### ISSUE TITLE
Optional query params with None not working

### ISSUE THREAD

**Issue by tuenut** [Maillol/aiohttp-pydantic#56]
Hello! I have code like below:
```python
class PaginationParams(Group):
    page: Optional[int] = 1
    page_size: Optional[int] = 100

class MyApiView(PydanticView):
    async def get(self, pagination: PaginationParams):
        ...
```

And it works perfectly, but if I change the default values to `None`, the view responds with validation error details in the response:
```
[{"type": "missing", "loc": ["page"], "msg": "Field required", "input": {}, "in": "query string"}, {"type": "missing", "loc": ["page_size"], "msg": "Field required", "input": {}, "in": "query string"}]
```

For pagination parameters, it is not critical, but for filter parameters, it may be necessary.

**Comment by Maillol** [comment]
Thanks for the bug report. It's fixed

**Comment by Maillol** [comment]
Released in version 2.1.1 

https://pypi.org/project/aiohttp-pydantic/2.1.1/
